### ISSUE TITLE
edit-to-testing-documentation

### ISSUE THREAD

**Issue by JaneSullivan-NOAA** [NOAA-FIMS/collaborative_workflow#83]
edit to testing documentation for setting up your path for [ninja](https://noaa-fims.github.io/collaborative_workflow/testing.html#adding-cmake-and-ninja-to-your-path-on-windows)

in addition to adding cmake to your path you also need to add path to ninja `C:\Users\Jane.Doe\Documents\Apps\ninja-win`

**Comment by JaneSullivan-NOAA** [comment]
@Bai-Li-NOAA or @kristanblackhart-NOAA hi there, it looks like i don't have permissions to create a new branch to address this edit. i've already created one locally and am trying to fetch it upstream but am getting denied:

```
$ git push -u origin 83-edit-testing-doc 

remote: Permission to NOAA-FIMS/collaborative_workflow.git denied to JaneSullivan-NOAA
```

**Comment by Bai-Li-NOAA** [comment]
Hi Jane, thanks for editing the testing document! I do not have access to add contributors to the repo, so I @ChristineStawitz-NOAA for help.

**Comment by iantaylor-NOAA** [comment]
The text can surely be improved, but I thought that this part was adequate: "Click New, add path to the location of the Ninja executable, if needed (e.g., Documents\Apps), and click OK."

**Comment by JaneSullivan-NOAA** [comment]
Documents\Apps isn't the path to the ninja exe. i'm trying to clarify a statement that resulted in two separate helpdesk tickets for me, not argue the fact that i should have interpreted it correctly in the first place.

**Comment by iantaylor-NOAA** [comment]
Sorry @JaneSullivan-NOAA, totally fair. Our documentation should not be leading to helpdesk tickets. Please do add the more complete and detailed path info and let me know if there's anything I can do to help.

In my case, I have a totally different location because I went with the "other preferred folder" option in the instructions about downloading cmake and ninja https://noaa-fims.github.io/collaborative_workflow/testing.html#setup-for-windows-users: 
> Download [ninja v1.10.2 (ninja-win.zip)](https://github.com/ninja-build/ninja/releases) and put the application to Documents\Apps or other preferred folder.

So, the corrected instructions should make clear that the goal is to provide the path all the way to the ninja.exe file, wherever it is located.

Could we also suggest that users run the "where" command to check that the path is working to help windows find the cmake.exe and ninja.exe files? For me, that looks like
```
C:\Users\ian.taylor>where cmake
C:\~\cmake\cmake-3.23.1-windows-x86_64\bin\cmake.exe

C:\Users\ian.taylor>where ninja
C:\~\ninja-win\ninja.exe
```

**Comment by Bai-Li-NOAA** [comment]
@iantaylor-NOAA, adding "where" command is a great idea. We can also use "which" command to check that the path is working on Mac and Linux.

**Comment by iantaylor-NOAA** [comment]
Resolved in PR #85 thanks to @JaneSullivan-NOAA.

@Bai-Li-NOAA and others, if you see further changes needed to this section, feel free to reopen this issue.
